### DISCUSSION TITLE
Scheduled Reminders: real-time alert when your review is dismissed

### DISCUSSION THREAD

**Discussion by rojlarge** [community/community#12655] [Other Feature Feedback, Questions, & Ideas]
The real-time alerts of scheduled reminders are awesome! I've been a big fan since the Pull Panda days and they get about 90% of the way to making my organization's development flow really smooth and seamless. 

However one gap I've encountered frequently is the lack of an alert when a review I've already submitted is dismissed. My team uses `CODEOWNERS` to automatically assign reviews to members of the appropriate team, and we also have our repos set up to dismiss stale reviews when new code is pushed. This works great for the most part, but as a reviewer I've often missed that I need to go back and review a PR after new changes until the author lets me know. It would be great if I could get a real-time alert through Scheduled Reminders when my review is dismissed so I know to go back and check the PR!

Thanks, and just let me know if there's already a way to accomplish this that I missed.

**Comment by aleksanb** [comment]
We need this in our company as well, some of our developers use the github slack integration, which does not notify when reviews are dismissed, leaving the poor code unreviewed.

**Comment by jcook-uptycs** [comment]
We use Danger.js and created a module to tag users with dismissed reviews. Not a perfect fix, but way better then what GitHub offers!

**Comment by georgemccabe** [comment]
+1
I encountered this situation as well. It would be nice to get reminders when a pull request that was previously approved but now requires an approval due to subsequent commits.
